packages/mui-base/src/Composite/Root/useCompositeRoot.ts

```diff
@@ -178,10 +229,7 @@ export function useCompositeRoot(params: UseCompositeRootParameters) {
 
             onActiveIndexChange(nextIndex);
 
-            // Wait for FocusManager `returnFocus` to execute.
-            queueMicrotask(() => {
-              elementsRef.current[nextIndex]?.focus();
-            });
```

Comment:
@atomiks This broke a RadioGroup test... https://app.circleci.com/pipelines/github/mui/base-ui/4867/workflows/dbe5234b-0110-4b20-8b7e-d8dfd310df29/jobs/34622/parallel-runs/0/steps/0-106
Any idea why?

Reply:
The test breaks but it still works in the browser, I've separated this for now